community-operators/eclipse-che/eclipse-che.7.0.0-beta-3.0.csv.yaml

```diff
@@ -0,0 +1,358 @@
+            displayName: OpenShift oAuth
+            path: auth.openShiftoAuth
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+          - description: TLS routes
+            displayName: TLS Mode
+            path: server.tlsSupport
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+        statusDescriptors:
+          - description: Route to access Eclipse Che
+            displayName: Eclipse Che URL
+            path: cheURL
+            x-descriptors:
+            - urn:alm:descriptor:org.w3:link
+          - description: Route to access Keycloak Admin Console
+            displayName: Keycloak Admin Console URL
+            path: keycloakURL
+            x-descriptors:
+            - urn:alm:descriptor:org.w3:link
+          - description: Eclipse Che server version
+            displayName: Eclipse Che version
+            path: cheVersion
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:label'
+          - description: The current status of the application
+            displayName: Status
+            path: cheClusterRunning
+            x-descriptors:
+              - 'urn:alm:descriptor:io.kubernetes.phase'
+  keywords:
+    - workspaces
+    - devtools
+    - developer
+    - ide
+    - java
+  displayName: Eclipse Che
+  provider:
+    name: Eclipse Foundation
+  maturity: stable
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: true
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  version: 7.0.0-beta-3.0
+  maintainers:
+    - email: eivantsov@redhat.com
+      name: Eugene Ivantsov
+  description: >
+    A collaborative Kubernetes-native development solution that delivers OpenShift workspaces and in-browser IDE for rapid cloud application development.
+    This operator installs PostgreSQL, Keycloak, and the Eclipse Che server, as well as configures all three services.
+
+    ## Pre-Reqs
+
+    In addition to a standard namespaced role, this operator may require a **ClusterRole** that allows the operator service account to:
+
+
+    * list, get, create, update, patch, watch and delete oauthclients at a cluster scope (the operator creates oAuthclient and OpenShift v3 identity provider in Keycloak to enable Login With OpenShift in Eclipse Che)
+
+
+    The operator service account will require extra privileges if you enable `auth.openShiftoAuth` which is false in CR template by default.
+    After the operator is installed, grant the **che-operator** service account such privileges:
+    When **auth.openShiftoAuth** is enabled:
+
+
+    ```
+
+    oc create clusterrole che-operator --resource=oauthclients --verb=get,create,delete,update,list,watch
+
+    oc create clusterrolebinding che-operator --clusterrole=che-operator --serviceaccount=${NAMESPACE}:che-operator
```

Comment:
Why can't this additional clusterrole be specified in the CSV's `clusterPermissions` section?

Reply:
and doesn't it represent some security risk ?